modules/apps/27-interchain-accounts/types/packet.go

```diff
@@ -54,7 +54,11 @@ func (iapd InterchainAccountPacketData) ValidateBasic() error {
 
 // GetBytes returns the JSON marshalled interchain account packet data.
 func (iapd InterchainAccountPacketData) GetBytes() []byte {
-	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&iapd))
+	res, err := json.Marshal(&iapd)
```

Comment:
I think we still need to use `ModuleCdc.MustMarshalJSON`. See my comment on the issue:
> Do not switch to standard json library as this changes the json econding due to the use of the enum on the Type field.

I think the ICS specs should be updated to indicate interchain accounts uses proto3 Json

Reply:
Oh. I see. Tysm